src/components/common/ListRow/index.js

```diff
@@ -0,0 +1,37 @@
+import { Fragment } from 'react';
+import cn from 'classnames';
+
+import './styles.css';
+
+const ListRow = ({
+  key,
+  classProps,
+  onClick,
+  disabled,
+  leftText,
+  name,
+  photo,
+  rightText,
+  icon,
+  showIcon,
+  children,
+}) => (
+  <Fragment key={key}>
+    <button
+      className={cn('list-row-container', { ...classProps, isEnabled: !classProps?.isDisabled })}
+      onClick={onClick}
+      disabled={disabled}
+    >
+      <span className="list-row-left-text">{leftText}</span>
```

Comment:
```suggestion
      {!!leftText && <span className="list-row-left-text">{leftText}</span>}
```
Don't you need to check for its existence?

Reply:
I actually want that space that occupies even though is empty. The style has flex something and leaves a space

Reply:
Yes, I saw it. I thought it wasn't on purpose